src/main/java/seedu/address/logic/commands/DeleteCustomerCommand.java

```diff
@@ -36,6 +38,8 @@ public DeleteCustomerCommand(Phone targetPhone) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Customer> lastShownList = model.getFilteredCustomerList();
+        List<Order> orderList = model.getFilteredOrderList();
+        model.updateFilteredOrderList(PREDICATE_SHOW_ALL_ORDERS);
```

Comment:
Instead of doing this, would it be possible to have a method, e.g. `getOrdersForCustomer` so that you don't have to change the filter all the time? I'm afraid similar bugs as the issue mentioned before may appear